backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -17,7 +17,7 @@ module FQName =
   /// The name of a thing in the package manager
   // TODO: We plan to use UUIDs for this, but this is a placeholder
   type Package<'name> =
-    { owner : string; modules : NEList<string>; name : 'name; version : int }
+    { owner : string; modules : NEList<string>; name : 'name; version : int } // NEList
```

Comment:
Are users allowed type `@paul.MyType`? Probably they should, even if we don't allow types in that namespace (which I actually think might be fine?).

Similarly, maybe we shouldn't assert so heavily on function (and other) name regexes?